src/Controls/src/Core/Platform/ShapeExtensions.cs

```diff
@@ -7,7 +7,7 @@ internal static class ShapeExtensions
 	{
 		internal static WindingMode GetPathWindingMode(this IDrawable drawable, IShapeView shapeView)
 		{
-			if (drawable is not ShapeDrawable || shapeView is null || shapeView.Shape is not Path path)
+			if (drawable is not ShapeDrawable || shapeView is null || shapeView?.Shape is not Path path)
```

Comment:
Is the `?` needed? My understanding is that if it reaches the last condition, it's because it passed the previous condition which is the null check; so when it reaches the last condition, we know the shapeView there is not null. Is there something I'm misunderstanding?

Reply:
Is not, removed.